addon/mixins/route.js

```diff
@@ -161,7 +159,9 @@ const RouteMixin = Mixin.create({
       totalPagesParam,
       _infinityModelName: modelName,
       extraParams: options,
-      content: A()
+      content: A(),
+      _routeAfterInfinityLoad: get(this, 'afterInfinityLoad'),
+      _routeInfinityModelLoaded: get(this, 'infinityModelLoaded')
```

Comment:
Would these be public?  (Sry not at my computer and don’t see where hey are used)

Reply:
Not intended to be, no. But I didn't specifically document public/private behavior here.